types/highlightjs.d.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+declare module 'highlight.js/lib/highlight.js';
```

Comment:
I think we don't need this concrete line, we should be using the proper types for that, the ones defined in the correct `@types` module, the problem was with the bindings, so all the other definitions are right.